students.txt

```diff
@@ -15,4 +15,7 @@ List of students
 8. Daria P. 
 9. Max R.
 10. Dima Ch.
-11. Evgeniy K.
+11. Evgeniy Kanev
+Hobbies: traveling (especially hiking and climbing), team sports (football). Like good books and good wine))
```

Comment:
Please use a list for your hobbies. 
Start each new hobby with a new line, with a '-' before it. Here's an example:
My hobbies:
 - Hobby 1
 - Hobby 2

Add at least 4 hobbies.